lib/client/http-tracker.js

```diff
@@ -74,7 +74,7 @@ HTTPTracker.prototype.scrape = function (opts) {
 
 HTTPTracker.prototype.destroy = function (cb) {
   var self = this
-  if (self.destroyed) return
+  if (self.destroyed) return cb && cb()
```

Comment:
No need for `cb &&`. You can see that we just call `cb()` directly below. This is an internal function, so we're always sure that it's called with a `cb` parameter.